odl/tomo/geometry/conebeam.py

```diff
@@ -329,7 +339,10 @@ def det_radius(self):
     @property
     def det_curve_radius(self):
         """Detector curve radius of this geometry."""
-        return self.__det_curve_radius
+        try:
+            return self.detector.radius
+        except AttributeError:
+            return None
```

Comment:
You can write this a bit more compactly as `return getattr(self.detector, 'radius', None)`. It's a bit easier on the eye.